### ISSUE TITLE
Dados sensíveis de conexão

### ISSUE THREAD

**Issue by alanfm** [FelipeSamuel/DB-PDO#1]
Você está usando dados sensíveis da aplicação na raiz do projeto e ainda pode ser visualizado pelo browser.

**Comment by FelipeSamuel** [comment]
Sim. Normalmente o acesso ao arquivo seria configurado no servidor, bloqueando o acesso direto.
